src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Startup.cs

```diff
@@ -158,20 +158,22 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 #endif
             app.UseStaticFiles();
 
-            app.UseRouting(routes =>
-            {
-                routes.MapRazorPages();
-#if (IndividualB2CAuth || OrganizationalAuth)
-                routes.MapControllers();
-#endif
-            });
-
             app.UseCookiePolicy();
 
+            app.UseRouting();
+
 #if (OrganizationalAuth || IndividualAuth)
             app.UseAuthentication();
 #endif
             app.UseAuthorization();
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapRazorPages();
+#if (IndividualB2CAuth || OrganizationalAuth)
+                routes.MapControllers();
```

Comment:
Is it still routes? I would have thought it would be `endpoints.MapControllers();` now.

Reply:
Yes, bad copy-pasto